parquet-column/src/main/java/org/apache/parquet/column/statistics/SizeStatistics.java

```diff
@@ -136,8 +136,10 @@ public SizeStatistics(
       List<Long> definitionLevelHistogram) {
     this.type = type;
     this.unencodedByteArrayDataBytes = unencodedByteArrayDataBytes;
-    this.repetitionLevelHistogram = repetitionLevelHistogram;
-    this.definitionLevelHistogram = definitionLevelHistogram;
+    this.repetitionLevelHistogram =
+        repetitionLevelHistogram == null ? Collections.emptyList() : repetitionLevelHistogram;
+    this.definitionLevelHistogram =
+        definitionLevelHistogram == null ? Collections.emptyList() : definitionLevelHistogram;
```

Comment:
Was it an actual defect that may cause issues? Shouldn't we backport this?

Reply:
I see. But shouldn't this be part of #3124 then?

Reply:
If these lines are moved to that PR, the current PR will fail due to NPE. However, I don't want to mix the CLI addition with the feature in that PR. I can create a separate PR to backport the fixes only.